pytorch_to_returnn/torch/nn/modules/variable.py

```diff
@@ -17,16 +17,25 @@ def __init__(self, param: Parameter):
     super(Variable, self).__init__()
     assert isinstance(param, Parameter)
     self.param = param
+    self.parent_mod = None
 
   def create_returnn_layer_dict(self, *inputs):  # ignore inputs
     return {"class": "variable", "add_batch_axis": False, "shape": self.param.shape}
 
   def make_output_tensor_from_returnn(self, inputs_flat: List[Tensor], layer: LayerBase) -> Tensor:
+    naming = Naming.get_instance()
+    values = self.param.detach().numpy()
+    if naming.import_params_from_torch_namespace and self.parent_mod:
+      if self.parent_mod[0].is_original_torch_module:
```

Comment:
Actually I'm not sure anymore if we really need this check here.

Reply:
Me neither. Since there is no clear argument for checking it, I removed it for. Or would you prefer to keep it?

Reply:
No, remove it.